ibm_catalog.json

```diff
@@ -0,0 +1,92 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-icd-elasticsearch",
+        "label": "Databases-for-Elasticsearch",
+        "product_kind": "solution",
+        "tags": [
+          "ibm_created",
+          "target_terraform",
+          "terraform",
+          "databases",
+          "solution"
+        ],
+        "keywords": [
+          "elasticsearch",
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution",
+          "elasticsearch secure",
+          "database",
+          "nosql"
+        ],
+        "short_description": "Creates and configures an instance of IBM Cloud Databases for Elasticsearch.",
+        "long_description": "This architecture supports creating and configuring an IBM Elasticsearch instance with KMS encryption and Context Based Restriction rules.",
+        "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-icd-elasticsearch/blob/main/README.md",
+        "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-icd-elasticsearch/main/images/elasticsearch_icon.svg",
+        "provider_name": "IBM",
+        "features": [
+          {
```

Comment:
Can we add a feature for KMS encryption?

Reply:
added